recipes/slearn/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "slearn" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+
+source:
+  fn: slearn-0.2.4.tar.gz
```

Comment:
You should be able to drop the `fn` line.

Reply:
Now it is removed.